src/main/java/mate/jdbc/controller/RegistrationController.java

```diff
@@ -40,10 +41,20 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         String password = req.getParameter("password");
         String confirmPassword = req.getParameter("confirm");
         String encrypt = HashUtils.encrypt(password);
+        String user = Role.USER.toString();
+        String admin = Role.ADMIN.toString();
         if (password.equals(confirmPassword) && !isEmpty(login, firstName, lastName, password)) {
-            Driver driver = new Driver(firstName, lastName, login, encrypt);
-            driverService.create(driver);
-            resp.sendRedirect("/car/add?" + DRIVER_ID + "=" + driver.getId());
+            if (req.getParameter("button").equals(user)) {
+                Driver driver = new Driver(firstName, lastName, login, encrypt);
+                driver.setRole(user);
+                driverService.create(driver);
+                resp.sendRedirect("/car/add?" + DRIVER_ID + "=" + driver.getId());
+            } else {
+                Driver driver = new Driver(firstName, lastName, login, encrypt);
+                driver.setRole(admin);
+                driverService.create(driver);
```

Comment:
@EliosPride extract common logic out of block "if". Use if-else only for redirecting